templates/contrib/facets/facets-item-list--range-slider.html.twig

```diff
@@ -38,11 +38,8 @@
     {{ items.value }}
 
     <div class="pips-preview">
-      {{ 'From'|t }}
-      <span class="pips-preview-min"></span>
-      {{ 'to'|t }}
-      <span class="pips-preview-max"></span>
-      {{ 'year'|t }}
+
+      {{ 'From <span class="pips-preview-min"></span> to <span class="pips-preview-max"></span> year'|t }}
```

Comment:
Bestaat er geen nettere manier om dit te doen? Zoals dit of is er geen ondersteuning voor placeholders in twig voor Drupal:

    {% set min = "<span class='pips-preview-min'></span>" %}
    {% set max = "<span class='pips-preview-max'></span>" %}

    {% trans %}
      From {{ min|raw }} to {{ min|raw }} year
    {% endtrans %}

Reply:
@MPParsley, laten we dit dan voorlopig hoe het is? ik vind niet meteen een betere werkbare oplossing.

Reply:
Tuurlijk, geen probleem.